conda_smithy/cli.py

```diff
@@ -212,13 +212,15 @@ def __call__(self, args):
 
         print("CI Summary for {}/{} (can take ~30s):".format(owner, repo))
         if args.travis:
-            ci_register.add_project_to_travis(owner, repo)
+            # travis-ci.com registration is automatic
```

Comment:
Technically this is not true for all orgs. We've enabled it for conda-forge.

Reply:
we've also changed all the code to do it for .com

Reply:
So we no longer have `.org` capability

Reply:
What I mean is that `.com` registration is automatic, because we have enabled it for conda-forge.

Reply:
Please push the changes that you want in that case, since I am not clear what you actually want here.